semantical/semantical_test.go

```diff
@@ -32,6 +32,19 @@ func TestValidNullLimit(t *testing.T) {
 	}
 }
 
+func TestValiDistinct(t *testing.T) {
```

Comment:
Is multiple distinct allowed? Can you add a test case to cover the expected behaviour for  `select distinct message , distinct author from commits` and `select distinct from commits` for example

Reply:
I started doing the code with that assumption, but looking again it seems like distinct should be a single word https://www.w3schools.com/Sql/sql_distinct.asp so I switch from `Distincts []string` to `IsDistinct bool`. Thoughts?

Reply:
I agree with you, I just would like to see a test making sure that is not acceptable to do  "select distinct message , distinct author from commits" . A test to make sure that the parser returns an error.

Reply:
Agree, added one that expects a parsing failure on double `distinct`